hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpcengine/nio/TlsNioAsyncSocket_LargePayloadTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.tpcengine.nio;
+
+import com.hazelcast.internal.tpcengine.ReactorBuilder;
+import com.hazelcast.internal.tpcengine.net.AsyncSocketBuilder;
+import com.hazelcast.internal.tpcengine.net.AsyncSocket_LargePayloadTest;
+import com.hazelcast.internal.tpcengine.net.DefaultSSLEngineFactory;
+
+public class TlsNioAsyncSocket_LargePayloadTest extends AsyncSocket_LargePayloadTest {
+
+    @Override
+    public ReactorBuilder newReactorBuilder() {
+        return new NioReactorBuilder();
+    }
+
+    @Override
+    protected void customizeSocketBuilder(AsyncSocketBuilder clientSocketBuilder) {
```

Comment:
I would rename this field to 'socketBuilder' since it is called for both client and server side sockets.